### ISSUE TITLE
Track max and min of tokens

### ISSUE THREAD

**Issue by osatien** [beeb/pancaketrade#66]
- [x] I have searched the [issues](https://github.com/beeb/pancaketrade/issues) of this repo and believe that this is not a duplicate.
- [x] I have read the Readme and believe that my question is not covered.

## Feature Request

As we are reading price every 5s can we have record of max and min during last x? Being x something configurable per token perhaps?

**Comment by beeb** [comment]
The only reasonable way to do this without becoming a charting software with a huge database of all the price history, would be to store the min and max price together with the times they occured inside the token record.

I have trouble to see how this would be more valuable than clicking one of the chart links and make your own personnalized assessment of the price movement.

What I want implement however, are price movement alerts as described in https://github.com/beeb/pancaketrade/issues/27 but I lack the time to implement new features these days.

**Comment by osatien** [comment]
The problem I see is that I don't know the price when the alert start. And
when it says going to buy order 24 would be nice to see the token the alert
price and the real in the exchange.


Good job in any case!!

I wanted to add support for two admins but having both message together
would be troublesome.

I have added support to apprise through env var so I can be messaged with
pushover when it buys or sells

**Comment by osatien** [comment]
Well I didn't have in mind any bd just something with min and max price in mem and reset every day at 0 but perhaps is related to price alert

**Comment by beeb** [comment]
What do you mean by "alert price"?

**Comment by osatien** [comment]
the price at which the alerts started

**Comment by beeb** [comment]
I don't know what you mean by "alert" since there is no alert feature as of yet. There are tokens and orders which react to price in various ways.

**Comment by osatien** [comment]
It's ok .... When I set limit order .... I put down prize and I am talking
about this signals. When I buy I have modified to send alert ... So this is
why I call them alerts

**Comment by beeb** [comment]
So if I understand correctly, you would like to see more information on one of the messages that are produced by the bot. Can you show which message (copy-paste or screenshot) you would like to change and what the new content should be?

**Comment by osatien** [comment]
🔸 Trying to buy for 0.2 BNB of NMS...

**Comment by beeb** [comment]
Yes, this message is shown while the transaction is pending. After that you should get a summary of which order executed, and then how much money you got from the sale, as well as the effective price (after any tax) that you got the tokens for. I really can't understand what you want, you need to be more clear. Probably a language barrier thing too. Try in spanish, I think I would understand because my mother is spanish.

**Comment by osatien** [comment]
I don't mind ... I know that I explain myself very bad and it should be the
same in Spanish 🤣

When it is ok no problem. But when the TX fails ... I don't know nothing.

And when the pool has high variability ... I don't know the price it was
